tests/e2e/tests/simple/a_simple_1_test.go

```diff
@@ -131,7 +131,7 @@ func TestSimpleIngress(t *testing.T) {
 func TestSvc2Svc(t *testing.T) {
 	ns := tc.Kube.Namespace
 	// Get the 2 pods
-	podList, err := getPodList(ns, "app=echosrv")
+	podList, err := getPodList(ns, "app.kubernetes.io/name=echosrv")
```

Comment:
For this test to pass "app" label needs to be changed to "app.kubernetes.io/name" in tests/simple/testdata/v1alpha3/servicesNotInjected.yaml and tests/simple/testdata/v1alpha3/servicesToBeInjected.yaml